src/components/CustomFeeModal/style.scss

```diff
@@ -0,0 +1,74 @@
+.custom-fee-modal {
+    .fee-modal-navbar {
+        background: var(--ion-color-primary);
+        border-bottom: 0px;
+        color: #ffffff;
+        width: 100%;
+        padding: 12px 16px;
+        display: flex;
+        align-items: center;
+        ion-icon {
+          width: 20px;
+          height: 20px;
+        }
+      }
+
+    ion-footer{        
+        --background: var(--ion-color-primary);
+        background: var(--ion-color-primary);
+        padding-top: 12px;
+        padding-bottom: 24px;
+        padding-left: 24px;
+        padding-right: 24px;
+    }
+
+    .fee-modal-content {
+        margin-top: 24px;
+        --padding-start: 0;
+        --padding-end: 0;
+        --overflow: auto;
+        --background: var(--ion-color-primary);
+
+        ion-item {
+          margin-top: 8px;
+          margin-bottom: 12px;
+          height: 66px;
+          display: flex;
+          justify-content: space-between;
+          margin-left: 24px;
+          margin-right: 24px;
+          border: 2px solid var(--ion-color-medium-tint);
+          color: var(--ion-color-tertiary);
+          &.active {
+              --background: --var(--ion-color-secondary-dark);
```

Comment:
`var` not `--var`

Reply:
Noted, made a misstep with the ionic syntax

Reply:
I meant that ionic uses a lot of custom properties. So I got carried away with using the "--" too much 😂. Like I said, **my mistake**.

Amusingly enough it still works as intended ( maybe that's why I didn't catch it)

![debug_color](https://user-images.githubusercontent.com/45250003/153932945-7aa32708-cd3c-4207-8ae9-712b4102178d.gif)

Reply:
Yes if works without it it's probably not necessary